src/coreclr/jit/emitarm64.cpp

```diff
@@ -5219,6 +5270,25 @@ emitter::code_t emitter::emitInsCodeSve(instruction ins, insFormat fmt)
     }
 }
 
+/*static*/ insOpts emitter::optWidenSveElemsizeArrangement(insOpts arrangement)
```

Comment:
Similar to above, I think this should be merged with `optWidenElemsizeArrangement()`

Reply:
Merging them would probably make it easier.

Reply:
same as above. keeping them separate will make sure it is get wrongly called.